frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```
@@ -131,6 +142,11 @@ action.GitMachete.FastForwardParentToMatchSelectedBranchAction.GitMacheteContext
 action.GitMachete.FastForwardParentToMatchSelectedBranchAction.description=Fast forward parent to match selected branch
 
 
+action.GitMachete.OverrideForkPointAction.text=Git Machete: Override Fork Point Of Selected Branch
+action.GitMachete.OverrideForkPointAction.GitMacheteContextMenu.text=O_verride Fork Point Of Branch
+action.GitMachete.OverrideForkPointAction.description=Override fork point of this branch
```

For more clarity, sth like:
```suggestion
action.GitMachete.OverrideForkPointAction.description=Select where the range of commits belonging to this branch starts
```